src/main/kotlin/racing/RaceCLIApplication.kt

```diff
@@ -0,0 +1,48 @@
+package racing
+
+import racing.domain.RaceProcessor
+import racing.infrastructure.RandomNumberGenerator
+import racing.presentation.RaceController
+import racing.presentation.request.RaceRequest
+import racing.view.ConsoleInputView
+import racing.view.ConsoleResultView
+
+object RaceCLIApplication {
+    private val inputView = ConsoleInputView()
+    private val resultView = ConsoleResultView()
+    private val numberGenerator = RandomNumberGenerator()
+    private val raceProcessor = RaceProcessor()
+    private val raceController =
+        RaceController(
+            raceProcessor = raceProcessor,
+            numberGenerator = numberGenerator,
+        )
+
+    @JvmStatic
+    fun main(args: Array<String>) {
+        val names = setupNames()
+        val tryCount = setUpTryCount()
+
+        val request = toRequest(names, tryCount)
```

Comment:
```suggestion
        val request = RaceRequest(names, tryCount)
```

감싸주지 않아도 좋다고 봐요 :) 객체 이름이 Request로 알 수 있어서 이미 명확하다고 생각해요 😊